src/rust/mqtt-publisher/src/main.rs

```diff
@@ -136,16 +137,26 @@ async fn eventpoll_loop(mut eventloop: EventLoop) -> Result<()> {
 struct PriceLevel {
     price: u128,
     size: u128,
+    txn_version: u128,
 }
 
 async fn price_level_loop(db_url: &str, mqtt_client: Arc<RwLock<AsyncClient>>) -> Result<()> {
     let pool = PgPool::connect(&db_url).await?;
 
     loop {
         tokio::time::sleep(Duration::from_millis(50)).await;
+        let mut tx = pool.begin().await?;
+        tx.execute("SET TRANSACTION ISOLATION LEVEL REPEATABLE READ;").await?;
```

Comment:
Do we already have a helper for repeatable read?

If so I suggest using that so that we are using consistent abstraction

Reply:
We have a helper function in the aggregator, so I can't use it here. One way to do this would be to create a new crate with just this function in it. I think this would just make everything more complex IMO.